docs/glossary.md

```diff
@@ -259,7 +269,19 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 #### [Multi-Cluster Replication](/self-hosted-guide/multi-cluster-replication)
 
-Multi-Cluster Replication is a feature which asynchronously replicates Workflow Executions from active Clusters to other passive Clusters, for backup and state reconstruction.
+Multi-Cluster Replication asynchronously replicate Workflow Executions and metadata from an active [Cluster](#temporal-cluster) to a standby Cluster.
+The replicated data is used for backup and state reconstruction.
+
+_Tags: [term](/tags/term), [explanation](/tags/explanation)_
+
+<span style={{ backgroundColor: 'rgb(255, 255, 102)', fontWeight: 'bold' }}>When MRN goes live update this link from cloud to cloud/multi-region</span>
+#### [Multi-region Namespace](/cloud)<!--/multi-region)-->
+
+A Multi-region Namespace (MRN) is a [Temporal Cloud](#temporal-cloud) Namespace that is configured to work across an active [region](/cloud/service-availability#regions) and a standby region located within the same continent.
+Each region is hosted on a separate Temporal Cloud [Cluster](#temporal-cluster). 
+Temporal Cloud automatically replicates Workflow Executions and metadata from the active to the standby region using [Multi-Cluster Replication](#multi-cluster-replication).
+MRNs are designed to respond to service issues like network congestion.
+When service to the primary Cluster is compromised, a [failover](#failover) transfers control from the active to the standby Cluster.
```

Comment:
Cluster -> Temporal Service

Above, "Failover" is defined exclusively for Multi-Cluster Replication - 
So, we want to expand Failover to include MRN.